lib/publiccloud/basetest.pm

```diff
@@ -35,6 +36,16 @@ sub provider_factory {
             subscription => get_required_var('PUBLIC_CLOUD_SUBSCRIPTION_ID')
         );
     }
+    elsif (check_var('PUBLIC_CLOUD_PROVIDER', 'GOOGLE')) {
```

Comment:
why GOOGLE in all caps? I know, someone wrote also AZURE but it does not look correct. EC2 is fine though :)

Reply:
I see the point regarding the CAPs. And I would keep it consistent.
Then let it cleanup in once, cause we also need to update the testsuites.

Reply:
shall we update it in this patch or you will take care of it later on?

Reply:
don't see any functional problems that it can bring. I think is discussion is too long for such minor thing => merging